packages/core/vind/_font.scss

```
@@ -0,0 +1,17 @@
+@use "../jkl";
+
+.font-sans {
+    @include jkl.use-font-family("Fremtind Grotesk");
+}
+
+.font-mono {
+    @include jkl.use-font-family("Fremtind Grotesk Mono");
+}
+
+.font-normal {
+    font-weight: jkl.$typography-weight-normal;
+}
+
+.font-bold {
+    font-weight: jkl.$typography-weight-bold;
+}
```

Burde vi "namespace" vind-klassene ved å wrappe dem i `.jkl {}`? Navnene er jo ganske generiske (og delvis delt med Tailwind), så ved å kun la dem fungere inne i elementer med klassen `jkl` kan vi potensielt unngå noen konflikter.

Alle Jøkul-produkter trenger uansett `jkl`-klassen ytterst for å fungere, så det blir ikke merarbeid på utviklerne som bruker det.

```suggestion
.jkl {
    .font-sans {
        @include jkl.use-font-family("Fremtind Grotesk");
    }

    .font-mono {
        @include jkl.use-font-family("Fremtind Grotesk Mono");
    }

    .font-normal {
        font-weight: jkl.$typography-weight-normal;
    }

    .font-bold {
        font-weight: jkl.$typography-weight-bold;
    }
}
```

---

Det kan vi godt. Ulempen er økt filstørrelse, men det komprimeres nok ganske greit bort.